### PR TITLE
Added a suggestion of solution with a simple for loop

### PR DESCRIPTION
**Pull request by tiagosalema**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by Dharmin2108** [comment]
xxx

**Comment by Dharmin2108** [comment]
hi

**Comment by Dharmin2108** [comment]
hh

**Comment by Dharmin2108** [comment]
nh

**Review by ojeytonwilliams — APPROVED** [review]
LGTM. @thecodingaviator Could you give this a look and merge it if you're happy?

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md [hunk 1]
```diff
@@ -187,6 +187,45 @@ Note: If the array only has two elements, then the `for` loop never gets used an
...
+          i = max;
+        }
+      }
+      return sol;
+    };
+    
+    // test here
+    smallestCommons([1,5]);
+    
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/@tiagosalema/smallestCommons' target='_blank' rel='nofollow'>Run Code</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]: Hi, thank you for your PR but now we are no longer accepting links or other resources (images, etc.) which are not under our control, i.e. which a user or service might remove and is not under our control.
Please remove this link.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md [hunk 2]
```diff
@@ -187,6 +187,45 @@ Note: If the array only has two elements, then the `for` loop never gets used an
...
+*   As a first guess, let's say that the solution is `max`. (we will increment this value later on if it is not the solution)
+*   Confirm that our solution is a multiple of all the values between `max` and `min` using a `for` loop.
+*   In case it isn't a solution, increment `max` to our solution (i.e. get the next multiple of the `arr` highest value) and restart the loop `i = max`. Note that it isn't `i = max - 1` since the `for` loop hasn't finished yet. Once it is finished, the loop itself will execute `i--`. It is also worth mentioning now that we started the loop at `i = max` and decremented `i` throughout instead of starting at `i = min` and then increment it in order to minimize the number of iterations.
+*   The `if` statement never being true means that all numbers between `min` and `max` are divisible by our solution
+
+#### Relevant Links
+
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Decrement_(--)' target='_blank' rel='nofollow'>Prefix decrement operator --</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]:
```suggestion
*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Decrement_(--)' target='_blank' rel='nofollow'>Prefix decrement operator (--)</a>
```

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md [hunk 3]
```diff
@@ -187,6 +187,45 @@ Note: If the array only has two elements, then the `for` loop never gets used an
...
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder_()' target='_blank' rel='nofollow'>Remainder operator --</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]:
```suggestion
*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder_()' target='_blank' rel='nofollow'>Remainder operator (%)</a>
```

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md [hunk 4]
```diff
@@ -187,6 +187,45 @@ Note: If the array only has two elements, then the `for` loop never gets used an
...
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min' target='_blank' rel='nofollow'>Math.min()</a>
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max' target='_blank' rel='nofollow'>Math.max()</a>
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax' target='_blank' rel='nofollow'>Spread syntax ...</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]:
```suggestion
*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax' target='_blank' rel='nofollow'>Spread syntax (...)</a>
```

**Review by thecodingaviator — DISMISSED** [review]

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]: Fixed on behalf of user.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md (hunk 2 above)
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]: Fixed on behalf of user.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md (hunk 3 above)
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]: Fixed on behalf of user.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md (hunk 4 above)
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/smallest-common-multiple/index.md]: Fixed on behalf of user.

**Comment by RandellDawson** [comment]
@thecodingaviator I made the changes you suggested.  Please merge if everything is good now.

Thanks.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Congrats @tiagosalema on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Review by thecodingaviator — COMMENTED** [review]
Good work!

**Pull request merged**